modules/ROOT/pages/index-secrets-manager.adoc

```diff
@@ -5,13 +5,52 @@ endif::[]
 
 Secrets Manager uses secure vault technology to store and control access to private keys, passwords, certificates, and other secrets.
 
-[NOTE]
-Secrets Manager is supported on Runtime Fabric and API Manager only. 
+Use Secrets Manager to write and manage your secrets, keys, and Transport Layer Security (TLS) artifacts. You can read the metadata of the secrets, but the actual secrets can be consumed only by authorized platform services on your behalf.
 
-Use Secrets Manager to write, read, and manage your secrets, keys, and Transport Layer Security (TLS) certificates within a unique source that allows access to other authorized platform services on your behalf.
+Secrets Manager is designed to store and manage secrets for supported Anypoint platform services. It is not a general-purpose storage for secrets. Only trusted services within Anypoint Platform have access to the contents of the secret. 
+
+
+Secrets Manager supports the management of TLS context for the following services:
+
+* Runtime Fabric ingress +
+You can store TLS artifacts in Secrets Manager and then configure the Runtime Fabric ingress with the secret reference.
+* API Manager in CloudHub +
+You can store the TLS artifacts in Secrets Manager and then configure the API Manager with the secret reference.
+
+The Anypoint platform’s Secrets Manager lets you store secrets in secret groups, which are vaults associated with your environment and business group. Each secret group has unique encryption keys, which are generated and managed only by Secrets Manager. 
+
+You can configure the supported Anypoint Platform services to request secrets stored in Secrets Manager, and you can control which supported services are authorized to access them. This task is handled by two microservices:
+
+* Secrets Manager +
+This is a publicly exposed service which handles the uploading and storing of your secrets.
+Every time you upload a secret to your vault, the Secrets Manager establishes a reference to it, so it can be shared or read without revealing its contents. You can read the metadata the secrets uploaded but not the real content. 
```

Comment:
```suggestion
Every time you upload a secret to your vault, the Secrets Manager establishes a reference to it, so it can be shared or read without revealing its contents. You can read the secrets metadata but not the secret itself. 
```